pkg/v21/key/key.go

```diff
@@ -173,7 +181,7 @@ func CommonChartSpecs() []ChartSpec {
 			ChannelName:     "0-5-stable",
 			ChartName:       "kubernetes-node-exporter-chart",
 			ConfigMapName:   "node-exporter-values",
```

Comment:
We are using cluster configmap, so we don't need to migrate this individual configmap.